packages/website/pages/docs/guides/run-a-node/run-a-taiko-node.mdx

```diff
@@ -145,6 +145,13 @@ For an Eldfell L3 node this would be:
 - `L2_ENDPOINT_HTTP`
 - `L2_ENDPOINT_WS`
 
+If you are running both L2 and L3 node , port will be 8545 for HTTP and 8546 for WSS.
+ex.)
+host.docker.internal:8545
```

Comment:
curious: are you using docker for mac/windows or docker for linux? curious if you had to add a:
```
extra_hosts:
    - "host.docker.internal:host-gateway"
```
as some linux users might need to for this to work.

Reply:
I use linux.
and what you refer to did not need to be set up.